client/signup/controller.js

```diff
@@ -284,25 +291,26 @@ export default {
 			flow: flowName,
 		} );
 
-		context.store.dispatch( setLayoutFocus( 'content' ) );
-		context.store.dispatch( setCurrentFlowName( flowName ) );
+		dispatch( setLayoutFocus( 'content' ) );
+		dispatch( setCurrentFlowName( flowName ) );
 
 		if ( ! [ 'launch-site' ].includes( flowName ) ) {
```

Comment:
from the discussion on this card, it sounds to me like adding `setup-site` to the array here would be acceptable, and wouldn't interfere with the use case in https://github.com/Automattic/wp-calypso/pull/42162